Makefile

```diff
@@ -1024,11 +1025,15 @@ publish-ci: image-ci
 # Docker image build for Teleport Operator
 .PHONY: image-operator-ci
 image-operator-ci:
-	make -C operator docker-build IMG=$(DOCKER_IMAGE_OPERATOR_CI):$(VERSION)
+	make -C operator docker-build IMG=$(DOCKER_IMAGE_OPERATOR_STAGING):$(VERSION)
 
 .PHONY: publish-operator-ci
 publish-operator-ci: image-operator-ci
-	docker push $(DOCKER_IMAGE_OPERATOR_CI):$(VERSION)
+	@if DOCKER_CLI_EXPERIMENTAL=enabled docker manifest inspect $(DOCKER_IMAGE_OPERATOR_STAGING):$(VERSION) 2>&1 >/dev/null; then\
+		echo "$(DOCKER_IMAGE_OPERATOR_STAGING):$(VERSION) already exists. ";     \
+	else                                                                         \
+		docker push $(DOCKER_IMAGE_OPERATOR_STAGING):$(VERSION);                 \
```

Comment:
Why not push every time? Won't redundant layers get filtered out anyway?

Reply:
The internal staging images use amazon ECR's immutable repository settings. This makes overwrites impossible currently. This can cause issues when drone tagging pipelines must be re-run due to failures. Currently the work around for this is to not attempt to push to the image when it already exists.

Reply:
Thanks for the explanation. That would be a good comment, if you want to sum it up a bit and add there.

Reply:
Added explanation in https://github.com/gravitational/teleport/pull/15275/commits/99a048997ab1a3b1637e89ffcc95476328a2ff5a

Reply:
Nice, thanks!